src/App.tsx

```diff
@@ -15,25 +15,36 @@ export function App(): JSX.Element | null {
       <TermContext.Provider value={{ term, setTerm }}>
         <Flex h="100vh" direction="column">
           <Header />
-          <Box grow={1} overflow="hidden" height="100%">
-            <Flex color="white" height="100%">
-              <SidebarContainer term={term as Term} pid={pid || ''} setPid={setPid} />
-              <Flex minW="80%" overflow="auto" justifyContent="center" height="100%" boxShadow="lg" zIndex={56}>
-                {pid ? (
-                  <Content term={term as Term} pid={pid} />
-                ) : (
-                  <Center p="10">
-                    <VStack>
-                      <Heading color="black">Explore Courses</Heading>
-                      <Text color="gray">
-                        Select a subject and then a course to start viewing course details and section information.
-                      </Text>
-                    </VStack>
-                  </Center>
-                )}
-              </Flex>
-            </Flex>
-          </Box>
+          <Router>
+            <Routes>
+              <Route path="/" />
```

Comment:
shouldn't the `/` route default to the app as we have now? As mentioned in the issue for this, theres no reason to make actual routes right now so I think only having this route, and the code in the route moved into a file in the `pages` folder (as discussed below) should suffice.

Reply:
yep! I could just remove that route, but i think @aomi wanted a "/" route to a blank page and a "/calendar" route to the homepage we have right now